enhancements/baremetal/continuous-hw-introspection.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Continuous HW introspecton of Baremetal IPI hosts
+authors:
+  - "@sadasu"
+reviewers:
+  - "@dhellmann"
+  - "@enxebre"
+  - "@hardys"
+  - "@stbenjam"
+  - "@juliakreger"
+approvers:
+  - "@dhellmann"
+  - "@enxebre"
+creation-date: 2020-02-18
+last-updated: 2020-02-17
+status: provisional|implementable|implemented|deferred|rejected|withdrawn|replaced
+---
+
+# Continuous HW introspection of Baremetal IPI hosts
+
+This proposal talks about how performing continuous HW introspection on all master and
+and worker nodes in a Baremetal IPI deployment would get us important data about the
+baremetal hosts which potentially can be used to monitor the health of individual nodes and
+the deployment as a whole.
+
+This enhancement will not result in a new CRD or API to be written but will detail
+data that would become availabe in the Status field of the BareMetalHost CRD.
+
+## Release Signoff Checklist
+
+- [X] Enhancement is `implementable`
+- [X] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+Continuous HW inspection on all the master and worker nodes that are a part of
+a Baremetal IPI deployment, allow the user to detect changes in server hardware
+and become aware of any alarms or failures seen on the servers. This information
+can be used to report important information to the user and/or glean metrics
+about the health and performance of the deployment.
```

Comment:
Could we explain what introspection is in more detail? And possibly link to Ironic documentation about it: https://docs.openstack.org/ironic-inspector/latest/

Reply:
Latest update has some more information about the hardware introspection. Please let me know if that amount works.